### PR TITLE
Space for file download button

### PR DESCRIPTION
**Pull request by SebaKas21**

### CODE REVIEW

**Review by dawiddworak88 — CHANGES_REQUESTED** [review]
HI @SebaKas21 , sorry I made wrong screenshot. Couldn't we implement it without using !important statements?

**Comment by SebaKas21** [comment]
@dawiddworak88 

I decided to create a class, because I had no idea, I used bulma but that I would not give me any space

**Review by dawiddworak88 — CHANGES_REQUESTED** [review]
Hi @SebaKas21 , unfortunately it still looks like on the picture below. We need to fix it differently, without using !important statements. They are bad practice and should be avoided. 

image

File: fe/src/project/AspNetCore/shared/components/Files/Files.scss [hunk 1]
```diff
@@ -0,0 +1,10 @@
+.file-tooltip {
```
Comment (dawiddworak88) [inline fe/src/project/AspNetCore/shared/components/Files/Files.scss]: **suggestion:** try to stick to BEM convention: http://getbem.com/naming/ It should be .files__tooltip

File: fe/src/project/AspNetCore/shared/components/Files/Files.scss [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+    margin-left: 0.5rem !important;
```
Comment (dawiddworak88) [inline fe/src/project/AspNetCore/shared/components/Files/Files.scss]: **suggestion:** It would be great to get rid of !important statements.

**Review by dawiddworak88 — CHANGES_REQUESTED** [review]

File: fe/src/project/AspNetCore/shared/components/Files/Files.scss (hunk 1 above)
Comment (SebaKas21) [reply fe/src/project/AspNetCore/shared/components/Files/Files.scss]: Thanks

**Review by dawiddworak88 — APPROVED** [review]
🥇

**Pull request merged**